### DISCUSSION TITLE
Button that acts like a link?

### DISCUSSION THREAD

**Discussion by emipallares** [adobe/react-spectrum#3459] [Q&A] [answered]
Hi,

In order to have a button that acts like a link (e.g. open a new tab to another page) that's what I'm currently doing:

```jsx
<a
    href="https://adobe.com"
    target="_blank"
    rel="noreferrer"
    style={{
        color: 'inherit',
        textDecoration: 'inherit',
    }}
>
    <Button variant="primary" UNSAFE_style={{ cursor: 'pointer' }}>
        Take the tour
    </Button>
</a>
```

Is there a way we could achieve the same behaviour (even with styles) with React Spectrum? I'm new to it and I was looking for something like a _LinkButton_.

Thanks!

**Comment by reidbarber** [comment, marked as answer]
This is actually already supported within Button:

```jsx
<Button variant="primary" elementType="a" href="https://adobe.com" target="_blank" rel="noreferrer">
  Take the tour
</Button>
```

**Reply by emipallares** [reply]
I did not notice this `elementType` prop. Thanks @reidbarber!
